deepmd/calculator.py

```diff
@@ -87,7 +87,9 @@ def __init__(
         **kwargs,
     ) -> None:
         Calculator.__init__(self, label=label, **kwargs)
-        self.dp = DeepPot(str(Path(model).resolve()), neighbor_list=neighbor_list)
+        self.dp = DeepPot(
+            str(Path(model).resolve()), neighbor_list=neighbor_list, **kwargs
+        )
```

Comment:
I am wondering if two `**kwargs` conflict. Besides, we don't have a doc for `**kwargs`.

Reply:
Extracting the `head` variable might improve clarity.

Reply:
Maybe we can make `head` an explicit argument as `neighbor_list` here.